spec/index.html

```diff
@@ -225,30 +301,29 @@
           attribute boolean? expanded;
           attribute boolean? disabled;
           attribute InvalidType? invalid;
-          attribute boolean hasPopUp;
+          attribute boolean? hasPopUp;
+          attribute boolean? pressed;
 
           // Control values.
-          attribute DOMString? value;
+          attribute DOMString? valueText;
```

Comment:
That was when we were hoping to expose the computed tree.
Maybe in the computed tree we can just expose one value.

Now that we've separated out the ideas of modifying accessibility
attributes in the DOM and getting their computed values, it makes
far more sense for Phase 1 to have parity with ARIA.

Parity with ARIA frees us up to not have to clearly define the semantics
for all of these attributes. Sure, it'd be nice to have a single "value", but
then what do you do if someone specifies both "value" and valueNow?
What about "value" and valueText?

My thinking now is that AccessibleNode in Phase 1 - 3 will be a strict superset of
ARIA, but then the computed accessible properties in Phase 4 can resemble the
native tree much more - i.e. "name", "description", etc.